### PR TITLE
Computer Science add EOC wrapper

### PR DESCRIPTION
**Pull request by LostMojo**
For [this Styles issue](https://github.com/openstax/cnx-recipes/issues/4011)

### CODE REVIEW

**Review by KarinaMendez2 — APPROVED** [review]

File: books/computer-science/bake [hunk 1]
```diff
@@ -48,7 +48,11 @@ recipe = Kitchen::BookRecipe.new(book_short_name: :computer_science) do |doc|
   book.chapters.each do |chapter|
     BakeLearningObjectives.v2(chapter: chapter, add_title: false)
     BakeNonIntroductionPages.v1(chapter: chapter)
-    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+
+    # create Chapter Review EOC wrapper
+    eoc_wrapper = ChapterReviewContainer.v1(chapter: chapter, metadata_source: metadata)
+
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata, append_to: eoc_wrapper)
     BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata, klass: 'section-summary')
```
Comment (kswanson33) [inline books/computer-science/bake:56]: should chapter summary also be appended to the wrapper?

**Review by kswanson33 — CHANGES_REQUESTED** [review]

File: books/computer-science/bake (hunk 1 above)
Comment (LostMojo) [reply books/computer-science/bake:56]: Good point-- the kitchen version doesn't currently have an append_to, so I'll make the change, and push that

Comment (KarinaMendez2) [reply books/computer-science/bake:56]: Maybe just use "Move custom section"?

File: kitchen_gem_versions.rb [hunk 2]
```diff
@@ -37,6 +37,6 @@
   psychology: '15.0.0',
   sociology: '7.0.0',
   statistics: '12.0.0',
-  world_history: '17.0.0',
+  world_history: '16.0.0',
```
Comment (kswanson33) [inline kitchen_gem_versions.rb:40]: why is wh being changed to 16?

**Review by kswanson33 — APPROVED** [review]

**Pull request merged**